src/App.js

```diff
@@ -1,12 +1,20 @@
 import './App.css';
+import Header from './components/Header/Header';
 import Calculator from './components/Calculator';
 import Quotes from './components/Quotes';
+import Home from './components/Home/Home';
+import { Routes, Route } from 'react-router-dom';
 
 function App() {
   return (
     <div className="app">
-      <Calculator />
-      <Quotes />
+ <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
+        <Route path="/quotes" element={<Quotes />} />
+      </Routes>
+      {/*  */}
```

Comment:
- [OPTIONAL] I suggest that you remove comments from your code this would ensure you have a cleaner codebase